datahub-web-react/src/app/entity/shared/containers/profile/header/PlatformContent/ParentNodesView.tsx

```diff
@@ -34,26 +34,27 @@ export const StyledTooltip = styled(Tooltip)`
     overflow: hidden;
 `;
 
-const GlossaryNodeText = styled(Typography.Text)<{ color: string }>`
-    font-size: 12px;
+const GlossaryNodeText = styled(Typography.Text)<{ color: string; fontSize: number }>`
     line-height: 20px;
+    font-size: ${(props) => props.fontSize};
     color: ${(props) => props.color};
 `;
 
-const GlossaryNodeIcon = styled(FolderOutlined)<{ color: string }>`
+const GlossaryNodeIcon = styled(FolderOutlined)<{ color: string; fontSize: number }>`
     color: ${(props) => props.color};
 
     &&& {
-        font-size: 12px;
+        font-size: ${(props) => props.fontSize};
         margin-right: 4px;
     }
 `;
 
 interface Props {
     parentNodes?: GlossaryNode[] | null;
+    customizeFontSize: number;
```

Comment:
let's make this prop optional, and then have a default in the styled components above in case we don't pass anything in. I don't want everyone to have to specify what the font size should be every time as opposed to just setting an override if it's so desired